chain/network/src/peer_manager/peer_manager_actor.rs

```diff
@@ -1722,3 +1727,34 @@ impl Handler<PeerManagerMessageRequest> for PeerManagerActor {
         self.handle_peer_manager_message(msg, ctx)
     }
 }
+
+impl Handler<NetworkDebugStatus> for PeerManagerActor {
+    type Result = Result<DebugStatusResponse, StatusError>;
```

Comment:
It seems to me that "type Result = DebugStatusResponse" would be enough here, since this call never returns an error.